packages/components/src/styles/components/layout/grid.scss

```diff
@@ -8,20 +8,22 @@
 //
 
 .hds-layout-grid {
+  // Note: "Unitless 0" <length>s aren’t supported in math functions so we use 0px as a default value within calc()
```

Comment:
[suggestion] maybe add a comment here, explaining why we need to initialize it here (instead of relying on the fallback in the `var()` declaration), the reason being to avoid being inherited from a parent value.

Reply:
I updated the comment with more info